mgmt/RecordsConfig.cc

```diff
@@ -1172,11 +1173,9 @@ static const RecordElement RecordsConfig[] =
   //# Configuration for TLSv1.3 and above
   //#
   //##############################################################################
-  // The default value (nullptr) means the default value of TLS stack will be used.
-  // - e.g. OpenSSL : "TLS_AES_256_GCM_SHA384:TLS_CHACHA20_POLY1305_SHA256:TLS_AES_128_GCM_SHA256"
-  {RECT_CONFIG, "proxy.config.ssl.server.TLSv1_3.cipher_suites", RECD_STRING, nullptr, RECU_RESTART_TS, RR_NULL, RECC_NULL, nullptr, RECA_NULL}
+  {RECT_CONFIG, "proxy.config.ssl.server.TLSv1_3.cipher_suites", RECD_STRING, "TLS_AES_128_GCM_SHA256:TLS_AES_256_GCM_SHA384:TLS_CHACHA20_POLY1305_SHA256", RECU_RESTART_TS, RR_NULL, RECC_NULL, nullptr, RECA_NULL}
```

Comment:
We prefer TLS_AES_256_GCM_SHA384 rather than TLS_AES_128_GCM_SHA256?
I'm not an expert here, but it might be better to use the default value that TLS stack have, like the removed comment says?

@masaori335 What do you think?

Reply:
I'm not sure we need to tweak default settings, because what is best is depends on use case and there'r no TLSv1.3 cipher suites which is too weak and we should remove from default settings.

Reply:
We run with AES128 in preference to AES256 for performance.  That is also the ordering for the non-TLS ciphers.

I am ok with going with AES256 first since this is just a default . In my opinion either ordering would be safe enough, but we should be consistent with the non-TLS1.3 list as well.  Going with AES256 first was the default for 8.x, and we had a rather lengthy discussion about the defaults for 7.x or 8.x.  Will update PR for that position.